code/game/objects/items/weapons/dice.dm

```diff
@@ -276,12 +296,11 @@
 			var/obj/effect/proc_holder/spell/summonmob/S = new
 			S.target_mob = H
 			user.mind.AddSpell(S)
-
 		if(17)
-			//Tator Kit
-			T.visible_message("<span class='userdanger'>A suspicious box appears!</span>")
-			new /obj/item/storage/box/syndie_kit/bundle(drop_location())
+			//Free Gun
+			T.visible_message("<span class='userdanger'>An impressive gun appears!</span>")
 			create_smoke(2)
+			new /obj/item/gun/projectile/revolver/mateba(drop_location())
```

Comment:
would like this changed to something weaker and laser-based. Possible options:

- Plasma Pistol
- Alien Pistol
- Ion Carbine
- PDW-9 Energy Pistol
- Advanced Energy Gun

This will ensure that miners don't just have easy access to a weapon that can hard crit two players nearly instantly.

Reply:
Added alien pistol in next commit